src/elem.js

```diff
@@ -265,6 +265,10 @@ exports.renderToStaticHtml = function(el) {
 
 exports.el = el;
 
+exports.jsx = function(name, attrs, children) {
```

Comment:
You're right, I've done this in es6...
To fix that, I just picked the code translated by babel.